incubator/stellar-core/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
```

Comment:
It's best to have two owners and maintainers.
Do you know anyone else who is familiar with the code and container you are using?
If not you could put me if you need it.